data_steward/utils/bq.py

```diff
@@ -240,3 +242,84 @@ def get_latest_validation_dataset_id(project_id):
             return sorted(validation_datasets, key=lambda x: x[0],
                           reverse=True)[0][1]
     return None
+
+
+def dataset_exists(dataset_id, project_id):
+    """
+    Checks if the dataset exists via dataset id
+
+    :param dataset_id:  ID (name) of the dataset to check
+    :param project_id:  ID (name) of the project to check for the dataset
+
+    :raises:  RuntimeError if a project_id is not supplied
+    :raises:  RuntimeError if a dataset_id is not supplied
+    :returns:  True if the dataset exists
+    :returns:  False if the dataset does not exist
+    """
+    if not project_id:
+        raise RuntimeError("Please specify a project")
+    if not dataset_id:
+        raise RuntimeError("Please specify a dataset to check if exists")
+    else:
+        client = get_client(project_id)
+        dataset_id = f"{project_id}.{dataset_id}"
+        dataset = bigquery.Dataset(dataset_id)
+
+        try:
+            client.get_dataset(dataset)
+            return True
+        except NotFound:
+            return False
```

Comment:
What is this providing that `create_dataset(dataset_ref, exists_ok=True/False)` doesn't?

Reply:
Nothing, it was suggested by @ksdkalluri to create this function for reasons I can't quite remember. @ksdkalluri can you elaborate on why I was to create this function instead of relying on the built in check in the client library `create_dataset`

Reply:
However, I have now added a check to make sure that `client` has been specified or else a RunTimeError is returned.

Reply:
@lrwb-aou @dcampbell-vumc sorry, I have missed the `exists_ok=False` in the `create_dataset()`. If that's available in the `create_dataset` function this function will be obsolete.

Reply:
@ksdkalluri .  this does exist and, if not explicitly provided, defaults to `exists_ok=False`